core/src/Platforms/Android/SystemWebview/AuthenticationActivity.cs

```diff
@@ -212,54 +207,26 @@ private string GetChromePackageWithCustomTabSupport(Context context)
             }
 
             Intent customTabServiceIntent = new Intent(_customTabsServiceAction);
-            IList<ResolveInfo> resolveInfoList = context.PackageManager.QueryIntentServices(
-                    customTabServiceIntent, 0);
+
+            IList<ResolveInfo> resolveInfoListWithCustomTabs = context.PackageManager.QueryIntentServices(
```

Comment:
>IList [](start = 12, length = 5)

Prefer IEnumerable as it is readonly by default. IList allows you to change the collection, which you do not need / want.

Reply:
great suggestion. fixed.

---
In reply to: [227735960](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1347#discussion_r227735960) [](ancestors = 227735960)